deploy_firestore_rules.sh

```
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Deploy Firestore Security Rules
+# This script deploys the Firestore security rules to Firebase
+
+echo "🚀 Deploying Firestore Security Rules..."
+
+# Navigate to the project directory
+cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)